### DISCUSSION TITLE
Completely unset numeric values

### DISCUSSION THREAD

**Discussion by de-es** [iSoron/uhabits#1722] [Feature Requests] [answered]
If I choose a wrong day for a numeric value by accident, I can't unset it. It would be nice if I could reset it back to "?".

Or is there a way to achieve this I couldn't find?

Tested in 2.1.1 and 2.1.2.

**Comment by chromer030** [comment, marked as answer]
Fixed : https://github.com/iSoron/uhabits/pull/1899

**Reply by de-es** [reply]
Unfortunately not yet released. I'll be patient. :grin:
